launcher/README.md

```diff
@@ -16,29 +16,14 @@ To run the notifier that pops up if `/proc/uptime` (how long the system has been
 
 ## Developer environment
 
-The next version of Qubes will include PyQt5 in dom0, which is why we also support PyQt5.
-
-### PyQt4 instructions
-
-In both cases, make sure to install the appropriate `python3-pyqt{4,5}` package in Debian Buster.
+Install the `python3-pyqt5` package in Debian Bullseye.
```

Comment:
Oh, here we go! : )

```suggestion
Install the `make`, `python3-venv`, `xvfb` and `python3-pyqt5` packages in Debian 11 (Bullseye), or `Xvfb` and `python3-PyQt5` in Fedora 36.
```

Reply:
Thank you - resolved in 8e15107, although I've avoided including Fedora instructions just cause I think it may muddy the waters a little.

Reply:
Sounds very fair 👍